.github/workflows/build_and_test.yml

```diff
@@ -27,10 +27,12 @@ jobs:
     - name: Pack APP
       run: rollup thoth-app/main.js --format iife --file thoth-app/pkg/thoth_app.js
     - name: Build binary
-      run: cargo build --verbose
+      uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --verbose
     - name: Run tests
-      run: cargo test --workspace --verbose
-    - name: Run linting
-      run: cargo clippy --all --all-targets --all-features -- -D warnings
-    - name: Check style
-      run: cargo fmt --all -- --check
+      uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --workspace --verbose
```

Comment:
Fix indentation - although the pull request check succeeded on the last commit, marking the PR ready to merge, the push check failed with syntax error (https://github.com/thoth-pub/thoth/actions/runs/592417934/workflow)